<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>LUCIO MARLON</author>
 <class>CadastrarPets</class>
 <widget class="QMainWindow" name="CadastrarPets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>636</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>622</width>
    <height>636</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>622</width>
    <height>636</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Pets</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>541</width>
      <height>581</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Cadastrar Pet</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>(G) - GATO - (C) - CACHORRO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCadastrarpet">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>490</y>
       <width>140</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 216, 98);
border-style: outset;
border-width: 2px;
border-radius: 15px; 
border-color: rgb(126, 126, 126);</string>
     </property>
     <property name="text">
      <string>Cadastrar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Porte</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ComboPorte">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPeso">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>420</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string>###,##</string>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
     <property name="placeholderText">
      <string>kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>420</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kg</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ComboRaca">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gato/Cachorro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>420</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Peso</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Espécie</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ComboEspecie">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="minimumContentsLength">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtIdade">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>81</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string>##,##</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Idade</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pet</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Responsável</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ComboNome">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNomepet">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>271</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>45</number>
     </property>
     <property name="placeholderText">
      <string>Nome do Pet</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <tabstops>
  <tabstop>ComboNome</tabstop>
  <tabstop>txtNomepet</tabstop>
  <tabstop>txtIdade</tabstop>
  <tabstop>ComboEspecie</tabstop>
  <tabstop>ComboRaca</tabstop>
  <tabstop>ComboPorte</tabstop>
  <tabstop>txtPeso</tabstop>
  <tabstop>btnCadastrarpet</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
